app/tests/spec/views/sign_up.js

```diff
@@ -1050,10 +1039,12 @@ define(function (require, exports, module) {
             });
         }
 
-        it('passes the customize sync option to the fxa-client', function () {
+        it('passes the expected options to `signUp`', function () {
           return setupCustomizeSyncTest('sync', true)
             .then(function () {
-              assert.isTrue(user.signUpAccount.args[0][0].get('customizeSync'), 'fxa client params');
+              assert.isTrue(view.signUp.calledOnce);
```

Comment:
Like the oauth test, this test duplicated work from the signup-mixin tests. Test up to the boundary, let the signup-mixin suite test the rest.